Mist/Helpers/Generator.swift

```diff
@@ -363,16 +388,46 @@ enum Generator {
             try updatePropertyList(url, key: "CFBundleShortVersionString", value: "12.6.03")
         }
 
-        var arguments: [String] = ["\(installer.temporaryInstallerURL.path)/Contents/Resources/createinstallmedia", "--volume", volume, "--nointeraction"]
-        let destinationURL: URL = .init(fileURLWithPath: volume).deletingLastPathComponent().appendingPathComponent("Install \(installer.name)")
+        var arguments: [String] = ["\(installer.temporaryInstallerURL.path)/Contents/Resources/createinstallmedia"]
+
+        if !installer.bigSurOrNewer {
+            // swiftlint:disable:next line_length
+            !options.quiet ? PrettyPrint.print("Copying '\(installer.temporaryInstallerURL.path)' to '\(installer.temporaryInstallerWithAdHocCodeSignaturesURL.path)'...", noAnsi: options.noAnsi) : Mist.noop()
+            try FileManager.default.copyItem(at: installer.temporaryInstallerURL, to: installer.temporaryInstallerWithAdHocCodeSignaturesURL)
+            !options.quiet ? PrettyPrint.print("Ad-hoc code signing '\(installer.temporaryInstallerWithAdHocCodeSignaturesURL.path)'...", noAnsi: options.noAnsi) : Mist.noop()
+            try adHocCodesign(installer.temporaryInstallerWithAdHocCodeSignaturesURL)
+            arguments = ["\(installer.temporaryInstallerWithAdHocCodeSignaturesURL.path)/Contents/Resources/createinstallmedia"]
+        }
+
+        arguments += ["--volume", installer.temporaryISOMountPointURL.path, "--nointeraction"]
```

Comment:
@ninxsoft I believe this broke `bootableinstaller`.

I think it's supposed to be:
``` suggestion
        arguments += ["--volume", volume, "--nointeraction"]
```